crates/swc_ecma_parser/tests/span/js/decl/namespace-export-string-names.js.spans

```diff
@@ -0,0 +1,48 @@
+warning: Module
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:1
+  |
+1 | export * as "忠恕。" from "Confucius"
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+warning: ModuleItem
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:1
+  |
+1 | export * as "忠恕。" from "Confucius"
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+warning: ModuleDecl
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:1
+  |
+1 | export * as "忠恕。" from "Confucius"
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+warning: NamedExport
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:1
+  |
+1 | export * as "忠恕。" from "Confucius"
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+warning: ExportSpecifier
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:8
+  |
+1 | export * as "忠恕。" from "Confucius"
+  |        ^^^^^^^^^^^^^
+
+warning: ExportNamespaceSpecifier
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:8
+  |
+1 | export * as "忠恕。" from "Confucius"
+  |        ^^^^^^^^^^^^^
+
+warning: Str
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:13
+  |
+1 | export * as "忠恕。" from "Confucius"
+  |             ^^^^^^^^
+
+warning: Str
+ --> $DIR/tests/span/js/decl/namespace-export-string-names.js:1:24
+  |
+1 | export * as "忠恕。" from "Confucius"
```

Comment:
Oh, span seems incorrect.

Reply:
Whats incorrect?

Reply:
![Screenshot_20220110-023022_Gmail.jpg](https://user-images.githubusercontent.com/29931815/148693475-077fb6c7-0ee8-4acd-90c6-2f43d6e037be.jpg)


It seems like a shifted span, although it might not be a fault of this PR

Reply:
Ah I got it. Thanks. I'll look into this.

Reply:
Probably, this isn't incorrect. Kanji-letters seem to correspond to two `^`.